app/src/modules/users/routes/item.vue

```diff
@@ -432,7 +432,7 @@ export default defineComponent({
 		async function deleteAndQuit() {
 			try {
 				await remove();
-				router.push(`/users`);
+				router.replace(`/users`);
```

Comment:
Lets only do it where it actually renders the detail page for a single item though. In these particular cases, you'd still want to have regular back/forth navigation.

Reply:
The changes I've made, are only in cases where the back stack is no longer available (usually due to deletion), `router.push()` still exists for majority of the pages.

Reply:
Oh I see, this is the detail page and on delete. Perfect! 👍🏻